docs/content/__internal__/index.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Internal
+caption: A page to test stuff
+---
+
+## Do's and Don'ts
+
+<Columns columns={[1, 1]} space={4}>
+  <Do>
+    <Do.Figure>
+      <Image
+        className="aspect-square"
+        width={700}
+        height={700}
+        unoptimized
+        src="https://upload.wikimedia.org/wikipedia/commons/1/18/Dog_Breeds.jpg"
+        alt="Don't"
+      />
+    </Do.Figure>
+    <Do.Description>
+      Use one primary call to action to help people to proceed
+    </Do.Description>
+  </Do>
+  <Dont>
+    <Do.Figure>
```

Comment:
Shouldn't there be used `Dont.Figure` and `Dont.Description`?

Reply:
is there a difference for them?

Reply:
Can't we just use `Figure` and `Description`?

Reply:
we could but then hey must be reusable
and yes it should be the dont figure!

Reply:
I don't understand, you did it accedently wrong and still do.description works for dont component :D